package.json

```diff
@@ -11,8 +11,8 @@
     "lint-all": "npm run lint-js && npm run lint-md && npm run lint-commit && npm run lint-css",
     "lint-staged": "lint-staged",
     "lint-css": "stylelint --syntax less components/css/**/*.less",
-    "dev-lint-js": "onchange components/** -i -- eslint --quiet --ext js --ext json --ext md --format html --output-file static/target/lint-out.html . --ignore-path .gitignore --ignore-pattern components/js/ga.js",
```

Comment:
I get the motivation behind pulling these patterns out into a file, but if the alternative is maintaining duplicated `.gitignore` and `.eslintignore`, I'd rather just add one more pattern here.

Reply:
#835 goes with adding another `--ignore-pattern` rather than introducing a `.eslintignore`.